main.cpp

```diff
@@ -52,106 +52,116 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: student
+//  action 1: the student study
+student.study();
+//  action 2: the student rest
+student.rest();
+//  action 3: the student sleep
+student.sleep();
 
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: employee
+//  action 1: the employee works
+employee.work();
+//  action 2: the employee eats
+employee.eat();
+//  action 3: the employee goes back home
+employee.goesBackHome();
 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: computer
+//  action 1: the computer calculates
+computer.calculate();
+//  action 2: the computer memorizes
+computer.memorize();
+//  action 3: the computer consumes energy
+computer.consumeEnergy();
+
+//  4) 
+//  Noun: car
+//  action 1: the car running
+car.running();
+//  action 2: the car consumes fuel 
+car.consumeFuel();
+//  action 3: the car close the door
+car.closeDoor();
```

Comment:
I'm marking this because:
1. your plain-english description is not an action, but is an ability or describes a state OR
2. the plain-english description is fine but the function name doesn't match OR
3. The way you wrote the name doesn't conform to the course coding standard:
    use `camelCase` for variable and function names.
    Type Names begin with a Capitalized first letter. All other symbols begin with a lower-case first letter.
- [ ] revise

Reply:
I followed the example: arm.flex(); then, i wrote: bird.fly().... how would be a correct form? and, name begining with a Capitalized don't follow de rule: camelCase.... camelCase begining with a lowercase and not whit a Capitalized letter... as I wrote above, in firstrevision, maybe I didn't understand something...

Reply:
car.closedDoor() it's following the example: arm.rotateCounterClockwise();  I can't see what's wrong....

Reply:
Type names with a Capitalized letter it's a ambiguos rule. if I follow the example: arm.flex, for example.... and it's ambiguos with camelCase to, that begining witha lowercase and upercase it's only that exists a secondterm in the name... How i explaned later, well..... I don't understand something......

Reply:
Sorry for my confusion... I must be wrong in something, but i don't know  what...